src/language/walker/syntaxWalker.ts

```diff
@@ -253,6 +265,15 @@ module Lint {
                     this.visitFunctionDeclaration(<ts.FunctionDeclaration> node);
                     break;
 
+                case ts.SyntaxKind.FunctionExpression:
+                    this.visitFunctionExpression(<ts.FunctionExpression> node);
+                    break;
+
+                case ts.SyntaxKind.GetAccessor:
```

Comment:
It is plasuible/likely the case that some future rule will want to walk these differently than regular method declarations. These should probably be their own functions -- though you can make a helper that all three call in your rule.